Configurations/windows-makefile.tmpl

```diff
@@ -88,12 +88,9 @@ DEPS={- join(" ", map { (my $x = $_) =~ s|\.o$|$depext|; $x; }
                   keys %{$unified_info{sources}}); -}
 {- output_on() if $disabled{makedepend}; "" -}
 GENERATED_MANDATORY={- join(" ", @{$unified_info{depends}->{""}} ) -}
-GENERATED={- join(" ",
-                  ( map { (my $x = $_) =~ s|\.[sS]$|\.asm|; $x }
-                    grep { defined $unified_info{generate}->{$_} }
-                    map { @{$unified_info{sources}->{$_}} }
-                    grep { /\.o$/ } keys %{$unified_info{sources}} ),
-                  ( grep { /\.(?:h|def)$/ } keys %{$unified_info{generate}} )) -}
+GENERATED={- # common.tmpl provides @generated
+             join(" ", map { (my $x = $_) =~ s|\.[sS]$|.asm|; $x }
```

Comment:
As you might have noticed (in #5929) I've started experimenting in complete .S-rule on Windows. Intention is that it should become handy on ARM. That means that .S files would have to be viewed as legitimate output for pl. Just in case for reference, I haven't modified it in #5929, because for far I was toying with already existing IA64 files. This is naturally a **side note**.

Reply:
I did notice your change, but was unsure where it was going, so...

Reply:
I'm thinking that this line will get modified when needed